### PR TITLE
Updated dates to fall 2021

### LINKED ISSUE #2: Update Semester, Year, and Date in Slides
Each lesson has still `SPRING 2020` and the old date of the spring semester of the JS course in Berlin.

Update to `FALL 2021` and each day of lesson of the semester.

### PR DESCRIPTION
**Pull request by RajshreeVats**
Close #2

### CODE REVIEW

File: lesson14.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <!-- .slide: id="lesson14" -->
 
-# Basic Frontend - Spring 2020
+# Basic Frontend - Fall 2021
 
-Lesson 14, Thursday, 2020-04-23
+Lesson 14, Thursday, 2021-10-04
```
Comment (lorepirri) [inline lesson14.md]: 2021-11-04

**Comment by lorepirri** [comment]
Hi @RajshreeVats thanks a lot for your contribution!

Sorry about the task, it is one of those repetitive tasks! Nevertheless, they are very useful to practice the logistic of contributing, using git, and interact with other fellows!

I marked a date which is not correct, but no worries: it's totally normal to make mistakes, and that's why code reviews are made. You could correct, commit, and push it again. Thanks!

_One note_: you might have noticed that this particular PR is not really pointing to the issue that would solve (the #2). For complex issues, the issue and its PR should be always be connected.

This help the future-you, and other devs working on the codebase to understand the motivations behind the changes, because the discussion is kept in the issue and not here.

To link the issue and the PR is very simple in GitHub: you have two options:
1. In the description of the PR add a `Close #2` text, and you will see on the right column, under _"Linked Issues"_ the issue number 2 mentioned.
2. Click on the _"Linked Issues"_ text on the right column, and select the related issue.

[Adding a linked issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) in this way (using the verb "Close", or "Fix"), will make another magic happen: as soon as the PR is approved and merged into the specified branch, the related issue will be automatically closed 😸 this is awesome.

_One other note_: after opening the PR, if you think you are done with the changes, you could add as a reviewer, the person who opened the ticket in the first place, or someone who is responsible for the particular area of code or documentation. If you are still working on a PR, please add the suffix `WIP` to the title: this will tell other developers or the reviewers, that the changes are still in progress (as in fact, WIP stands for Work In Progress).

Could you please, when and if you have time:
1. Add the linked issue, preferably in the description with a `Close #2` text.
1. Fix the wrong date (see the other brief comment).
1. Ask me to review the changes once you're done.

Should you have lost patience for the task, of course I could do all of the above: I am just nitpicking now to let you practice ^^ just ping me 😄 

If you have questions, please drop a comment down below! 

Happy coding! 🌈

**Comment by RajshreeVats** [comment]
Thanks for these really helpful tips, will follow and practice.

File: lesson14.md (hunk 1 above)
Comment (lorepirri) [reply lesson14.md]: super! thanks!

**Comment by lorepirri** [comment]
super cool! I saw the changes, all good, I apologize I said issue number 3 while it was number 2 ^^ I've updated the description, and I'm gonna merge this!

Congratulation! Your first contribution to this repo looks awesome, precise and fast! Thanks a lot!

**Comment by lorepirri** [comment]
@RajshreeVats your changes are already available online here: https://redi-school.github.io/nrw-js/ 

it's a good feeling isn't it? Thanks again and happy coding!

**Comment by RajshreeVats** [comment]
Yes, it is. Thanks for your support in the process. : )

**Pull request merged**